ng-dev/pr/common/fetch-pull-request.ts

```diff
@@ -25,8 +41,28 @@ const PR_SCHEMA = {
       nodes: [
         {
           commit: {
-            status: {
-              state: graphqlTypes.oneOf(['FAILURE', 'PENDING', 'SUCCESS'] as const),
+            statusCheckRollup: {
+              state: graphqlTypes.custom<StatusState>(),
+              contexts: params(
+                {last: 100},
```

Comment:
What are we doing about the 100 here? should we add pagination? It's unlikely to hit 100, but it seems like this is a place where we _should_ be correct (as it's a common utility now)

Reply:
We should probably address it, but I think it makes sense in a follow up as it will be a bit complicated to do.

Reply:
Works for me 👍